src/arrayMethodSort.js

```diff
@@ -4,9 +4,17 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (callback = (a, b) => String(a).localeCompare(String(b))) {
     // write code here
-  };
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (callback(this[i], this[j]) > 0) {
+          [this[i], this[j]] = [this[j], this[i]]
```

Comment:
Using array destructuring for swapping elements is a modern and clean approach. Just ensure that the environment where this code will run supports ES6 features.